src/views/repository/content/router/RouterTuningsAdvanced.vue

```diff
@@ -167,6 +168,12 @@ export default {
 <style lang="scss" scoped>
 @import '~@weni/unnnic-system/src/assets/scss/unnnic.scss';
 
+.custom-switch :deep() {
+  span {
```

Comment:
I'm not sure if this works, could you validate if is it working?
I think the deep style logic should be inside eg. `.custom-switch :deep(span)`
https://vuejs.org/api/sfc-css-features.html#deep-selectors

Reply:
interesting, I'll try it